### PR TITLE
Highlight links in schema documentation

### LINKED ISSUE #6945: Config blocks in the docs don't properly indicate where any hyperlinks are
Screenshot 2024-04-01 at 11 42 21 AM

For example, from https://docs.taskcluster.net/docs/reference/core/worker-manager/google#worker-pool-configuration, the word `here` is linked to https://cloud.google.com/compute/docs/reference/rest/v1/instanceTemplates but it's not at all apparent you can click on that word (highlighted, underlined, change when your mouse hovers over top, etc.).

### PR DESCRIPTION
**Pull request by lotas**
image

Fixes #6945

### CODE REVIEW

File: changelog/issue-6945.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+audience: users
+level: minor
```
Comment (matt-boris) [inline changelog/issue-6945.md:2]: Wdyt?

```suggestion
level: patch
```

**Review by matt-boris — COMMENTED** [review]
Thanks for tackling so fast :) Left one quick suggestion on changelog

File: changelog/issue-6945.md (hunk 1 above)
Comment (lotas) [reply changelog/issue-6945.md:2]: lol, it was a long weekend, i forgot what levels we have 😆

**Review by matt-boris — APPROVED** [review]
🎉

**Pull request merged**